.npmignore

```diff
@@ -1 +0,0 @@
-src/
```

Comment:
What's the reason we're ignoring `src` again? Just so we don't have Babel as an install-time dependency?